client/src/mvc/collection/ListCollectionCreatorModal.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+import CollectionCreatorModalMixin from "./mixins/CollectionCreatorModalMixin";
+import _l from "utils/localization";
+import Vue from "vue";
+export default {
+    mixins: [CollectionCreatorModalMixin],
+    methods: {
+        listCollectionCreatorModal: function (elements, options) {
+            options = options || {};
+            options.title = _l("Create a collection from a list of datasets");
+            const { deferred, creatorOptions, showEl } = this.collectionCreatorModalSetup(options); // eslint-disable-line no-unused-vars
+            return import(/* webpackChunkName: "ListCollectionCreator" */ "./ListCollectionCreator.vue").then(
```

Comment:
Import at the top file like any other component. Then just do a normal Vue.extend.

Reply:
Is this actually breaking?  This isn't unreasonable if the ListCollectionCreator component is medium to large (including dependencies) and/or not used very frequently.

Reply:
I just assumed since it didn't have external dependencies relative to the rest of the app this wasn't needed. Rule builder made sense.